### PR TITLE
frontend/fix: #1547 IOS Release

### PR DESCRIPTION
**Pull request by MercyQueen**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### CODE REVIEW

File: Frontend/ui-customer/src/Strings/HI.js [hunk 1]
```diff
@@ -429,7 +429,7 @@ const hindiStrings = {
   "NO_DONT": "नहीं",
   "YES_CANCEL_SEARCH": "हाँ, खोज रद्द करें",
   "TRY_LOOKING_FOR_RIDES_AGAIN": "ऐसा लगता है कि यह बहुत ही व्यस्त दिन है। आप फिर से राइड खोजने की कोशिश कर सकते हैं",
-  "NO_TIP": "कोई टिप नहीं",
+  "NO_TIP": "नो टिप",
```
Comment (ashishsingh101) [inline Frontend/ui-customer/src/Strings/HI.js:432]: previous one was correct

File: Frontend/ui-customer/src/Strings/HI.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ const hindiStrings = {
   "TRIP_CHARGES": "यात्रा शुल्क",
   "PROMOTION": "पदोन्नति",
   "SEND_EMAIL": "ईमेल भेजें",
-  "YOU_CAN_DESCRIBE_THE_ISSUE_YOU_FACED_HERE": "आप यहां जिस समस्या का सामना कर रहे हैं उसका वर्णन कर सकते हैं",
+  "YOU_CAN_DESCRIBE_THE_ISSUE_YOU_FACED_HERE": "आप यहां सामने आई समस्या का वर्णन कर सकते हैं",
```
Comment (ashishsingh101) [inline Frontend/ui-customer/src/Strings/HI.js:24]: can you edit this

File: Frontend/ui-customer/src/Services/Config.purs [hunk 3]
```diff
@@ -51,7 +51,7 @@ getConfig = do
         , fingerprint : ""
         }
     DEV  -> Config
-        { baseUrl: getValueToLocalNativeStoreConfig "BASE_URL"
+        { baseUrl: "https://api.sandbox.beckn.juspay.in/dev/app/v2"
```
Comment (ashishsingh101) [inline Frontend/ui-customer/src/Services/Config.purs:54]: remove this

File: Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/View.purs [hunk 4]
```diff
@@ -379,7 +380,38 @@ editTextView state push =
...
+              , gravity CENTER_VERTICAL
+              , padding $ PaddingLeft 10
+              , margin $ MarginTop 5
+              , visibility if DS.length state.data.description < 10 then VISIBLE else GONE
+              ][  imageView $
+                  [ width $ V 16
+                  , height $ V 16
+                  , padding $ Padding 0 5 0 3
+                  , margin $ MarginRight 5
+                  , imageWithFallback "ny_ic_info,https://assets.juspay.in/nammayatri/images/user/ny_ic_information_grey.png"
```
Comment (ashishsingh101) [inline Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/View.purs:401]: can you validate this url again

File: Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/View.purs [hunk 5]
```diff
@@ -379,7 +380,38 @@ editTextView state push =
...
+              , width MATCH_PARENT
+              , orientation HORIZONTAL
+              , gravity CENTER_VERTICAL
+              , padding $ PaddingLeft 10
+              , margin $ MarginTop 5
+              , visibility if DS.length state.data.description < 10 then VISIBLE else GONE
+              ][  imageView $
+                  [ width $ V 16
+                  , height $ V 16
+                  , padding $ Padding 0 5 0 3
```
Comment (ashishsingh101) [inline Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/View.purs:399]: use padding vertical

File: Frontend/ui-customer/src/Flow.purs [hunk 6]
```diff
@@ -36,7 +36,7 @@ import Debug (spy)
 import Effect (Effect)
 import Effect.Class (liftEffect)
 import Engineering.Helpers.BackTrack (getState)
-import Engineering.Helpers.Commons (liftFlow, os, getNewIDWithTag, bundleVersion, getExpiryTime,stringToVersion, convertUTCtoISC, getCurrentUTC)
+import Engineering.Helpers.Commons (liftFlow, os, getNewIDWithTag, bundleVersion, getExpiryTime,stringToVersion, convertUTCtoISC, getCurrentUTC, clearTimer)
```
Comment (ashishsingh101) [inline Frontend/ui-customer/src/Flow.purs:39]: remove clearTimer

**Review by ashishsingh101 — CHANGES_REQUESTED** [review]

File: Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/View.purs (hunk 4 above)
Comment (MercyQueen) [reply Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/View.purs:401]: its working

File: Frontend/ui-customer/src/Strings/HI.js (hunk 1 above)
Comment (MercyQueen) [reply Frontend/ui-customer/src/Strings/HI.js:432]: Discussed this with Sreedeep.As the UI was breaking because of the text so we changed it.

**Review by aryan-13 — APPROVED** [review]

**Review by ashishsingh101 — APPROVED** [review]

**Pull request merged**